packages/lrod/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=lrod
+pkgver=Version.r32.g57d9cab
+pkgrel=1
+pkgdesc='Perform subdomain enumeration, endpoint recognition, and more.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon')
+url='https://gitlab.com/Edu0x01/Lrod'
+license=('MIT')
+depends=('python' 'python-virtualenv')
+makedepends=('git' 'python-setuptools' 'python-pip')
+source=("git+https://gitlab.com/Edu0x01/$pkgname.git")
+sha512sums=('SKIP')
+install="$pkgname.install"
```

Comment:
Your PR misses the install file https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/python-standalone.install

Reply:
Yes. Sorry, I didn't added it on the commit.